tooling/nargo/src/artifacts/debug_vars.rs

```diff
@@ -0,0 +1,132 @@
+use acvm::brillig_vm::brillig::Value;
+use noirc_printable_type::{decode_value, PrintableType, PrintableValue};
+use std::collections::{HashMap, HashSet};
+
+#[derive(Debug, Default, Clone)]
+pub struct DebugVars {
+    id_to_name: HashMap<u32, String>,
+    active: HashSet<u32>,
+    id_to_value: HashMap<u32, PrintableValue>, // TODO: something more sophisticated for lexical levels
```

Comment:
A Vec of scopes perhaps?

Reply:
Yes, lexical scopes are handled in #4188. But I see we haven't put a proper name for this there either. I'll change this along with changing the `u32` to more descriptive type names.

Reply:
This was changed to use proper identifier types. Lexical scope handling is left for #4188.